featuretools/computational_backends/feature_set_calculator.py

```diff
@@ -107,18 +107,18 @@ def progress_callback(*args):
         df_trie = Trie(path_constructor=RelationshipPath)
         full_entity_df_trie = Trie(path_constructor=RelationshipPath)
 
-        target_entity = self.entityset[self.feature_set.target_eid]
+        target_dataframe = self.entityset[self.feature_set.target_eid]
         self._calculate_features_for_entity(entity_id=self.feature_set.target_eid,
                                             feature_trie=feature_trie,
                                             df_trie=df_trie,
                                             full_entity_df_trie=full_entity_df_trie,
                                             precalculated_trie=self.precalculated_features,
-                                            filter_variable=target_entity.index,
+                                            filter_variable=target_dataframe.index,
```

Comment:
This will need to be `target_dataframe.ww.index`

Reply:
fixed